nixos/modules/services/continuous-integration/buildbot/master.nix

```diff
@@ -66,13 +66,10 @@ in {
       };
 
       masterCfg = mkOption {
-        type = types.str;
-        description = ''
-          Optionally pass raw master.cfg file as string.
-          Other options in this configuration will be ignored.
-        '';
+        type = types.path;
+        description = "Optionally pass master.cfg path. Other options in this configuration will be ignored.";
         default = null;
-        example = "BuildmasterConfig = c = {}";
+        example = /etc/nixos/buildbot/master.cfg;
```

Comment:
You'll want this to be quoted if it's an example, otherwise Nix will try to find this file when it evaluates the example (mostly when we build documentation)